app/workers/facilities/facility_location_download_job.rb

```diff
@@ -6,6 +6,7 @@ class FacilityLocationDownloadJob
 
     def perform(type)
       @type = type
+      Raven.tags_context(job: "FacilityLocationDownloadJob:#{type}")
```

Comment:
👍 this will make tracking down these types of failures easier. I propose we do something similar for other workers or jobs as well.